app/Moldova/Service/Dashboard.php

```diff
@@ -44,6 +44,18 @@ public function importData()
     {
         if (!$this->importStatus()) {
             dispatch(new ImportData());
+
+            //clear all cache after new data has been imported
+            if (Cache::has('routes')) {
```

Comment:
The job is just dispatched here and is not completed, so clearing of cache should be done after the completion of the import job.